Sources/ReactiveMoya/SignalProducer+Response.swift

```diff
@@ -14,18 +14,26 @@ extension SignalProducerProtocol where Value == Response, Error == MoyaError {
         }
     }
 
+    /// Filters out responses that don't fall within the given range, generating errors when others are encountered.
+    public func filter(statusCodes: Range<Int>) -> SignalProducer<Value, MoyaError> {
+        return filter(statusCodes: statusCodes.lowerBound...statusCodes.upperBound-1)
```

Comment:
this shouldn't have to use logic again, it is defined on `Response` and this way we can have inconsistent condense whenever we forgot to change the same line in 3 places after a fix/change... let's change it to:
```swift
     public func filter(statusCodes: Range<Int>) -> SignalProducer<Value, MoyaError> {
         return producer.flatMap(.latest) { response -> SignalProducer<Value, Error> in
             return unwrapThrowable { try response.filter(statusCodes: statusCodes) }
          }
      }
```

Reply:
@sunshinejr  Sure :))